kettocritic/serializers.py

```diff
@@ -27,16 +27,19 @@ def get_serialized_model(self, instance):
 
 
 class GameSerializer(ModelSerializer):
-    fields = ['name']
+    fields = ['id', 'name']
 
 
 class TeamSerializer(ModelSerializer):
-    fields = ['name', 'website']
+    fields = ['id', 'num_reviews', 'num_reviewers']
```

Comment:
does the serializer automatically grab the `num_reveiwers` property from the model or do we have to create a method for it?  wondering if we should just define the logic for the calculated property in the model and and grab it there.

Reply:
i think we need to define the logic for getting it from the model property